src/quantum/azext_quantum/_help.py

```diff
@@ -27,6 +31,10 @@
       - name: Submit the Q# program from the current folder and wait for the result.
         text: |-
             az quantum run -g MyResourceGroup -w MyWorkspace -l MyLocation -t MyTarget
+      - name: Submit and wait for a Q# program from the current folder with job and program parameters.
+        text: |-
+            az quantum run -g MyResourceGroup -w MyWorkspace -l MyLocation -t MyTarget \\
+                --job-params key1=value1 key2=value2 -- --n-qubits=3
```

Comment:
[2:20 PM] Yu Chen
It seems the help is not consistent with the example in help it says "specified as --name=value after --", but in example, it uses as "-- --n-qubits 3" (not --name=value format)


![image](https://user-images.githubusercontent.com/16348853/134692095-6e7bda82-606e-45be-a142-22c1c73b4b05.png)
![image](https://user-images.githubusercontent.com/16348853/134692132-58949f0c-3a03-4b3c-a7f2-9c1c146e9e26.png)

Reply:
This has been fixed in a more recent update, thanks Yu!